docs/extending/developing-code/extending/case-studies/Curl/in-mathics.rst

```diff
@@ -0,0 +1,156 @@
+Writing Curl in Mathics
+=======================
+
+.. contents::
+   :depth: 1
+   :local:
+
+Mathics is extensible; it allows creation of new functions, and new builtin functions all using the Mathics language.
```

Comment:
Shouldn't it be a "," instead of ";"?

Reply:
No I don't believe so. A semicolon is used to separate to complete sentence. It is almost the same as two sentences. But when one sentence is short or the two sentences can be considered tied together, a semicolon is sometimes used instead of a period and a new sentence.

On the other hand, a comma is used to separate phrases, of a single sentence.